### DISCUSSION TITLE
Phishing Website Targetting BNI (Bank Negara Indonesia)

### DISCUSSION THREAD

**Discussion by DsansCore** [vercel/community#5522] [Help]
### Summary

To Whom It May Concern,

I'm from copyright owner you can check from my email domain. This is for our trademark BNI. 
I am writing to report a phishing project that is using your servers. The project is using multiple free-service of the domain "vercel.app".

I have conducted a thorough investigation of the project and have found the following evidence that it is a phishing project:

The website uses a design that is similar to the design of a legitimate website.
The website requests personal information from users, such as usernames, passwords, and credit/debit card numbers.

The project uses the following subdomains:

free-poin-hadiah.vercel.app
bnlrewardpoin.vercel.app
view-hadiah.vercel.app

Here are the images example:

![Screen Shot 2024-01-19 at 14 53 34](https://github.com/vercel/community/assets/30978548/6600d04b-7ea3-47bb-a918-914d55accdda)
![Screen Shot 2024-01-19 at 14 54 05](https://github.com/vercel/community/assets/30978548/191f7d0e-af4a-40b0-9d6d-020067e96446)
![Screen Shot 2024-01-19 at 14 54 50](https://github.com/vercel/community/assets/30978548/1218b36f-ce14-4936-a247-5853c990ef38)

I am concerned that this website is posing a serious threat to the security of your users. I urge you to immediately suspend the website and take steps to prevent it from being used to steal personal information from users.

### Example

free-poin-hadiah.vercel.app

### Steps to Reproduce

We need the account or the site that creates the phishing website using vercel's free service site to be taken down, so none of the customers private information are being harm. We tried contacting the abuse@vercel.com to take down the site, but the progress is so slow, and we need it down as soon as possible.

To contact us you can email us here
cti@bni.co.id

Thank you for your time and attention to this matter.

Best Regards,
#DRH

**Comment by tiarasakinah** [comment]
i almost got tricked by these urls

**Reply by DsansCore** [reply]
Really? I tried messaging abuse@vercel.com but they didn't share any progress with the takedown...
